server/migrations/20240327015248-add-user-role.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    await queryInterface.addColumn("users", "role", {
+      type: Sequelize.ENUM("admin", "member", "viewer", "guest"),
```

Comment:
ENUM's are annoying to migrate so including `guest` here from the beginning, although it won't be used yet.